tests/php/RestApi/StoreApi/Controllers/Cart.php

```diff
@@ -108,4 +113,18 @@ public function test_prepare_item_for_response() {
 		$this->assertArrayHasKey( 'items_weight', $response->get_data() );
 		$this->assertArrayHasKey( 'totals', $response->get_data() );
 	}
+
+	/**
+	 * Test schema matches responses.
+	 */
+	public function test_schema_matches_response() {
```

Comment:
Love this! Do we have any setup instructions for running the PHP tests? If not we should add this to a readme, or even a recommended/default dev environment.

I installed PHPUnit in my environment (inside docker container) but then hit issues with missing core tests, which required svn to be installed ..

`Could not find /tmp/wordpress-tests-lib/includes/functions.php, have you run bin/install-wp-tests.sh ?root@d54ab9759684:/var/www/html/wp-content/plugins`

Reply:
Can't help you with Docker sorry :) I use VVV where, after ensuring composer has installed, you SSH in and run from the plugin directory with `﻿.vendor/bin/phpunit`

Reply:
Confirmed I'm running the right phpunit now, still getting the same error. I'm guessing our tests depend on some base tests from WordPress core. I'm guessing this is set up in your VVV, so perhaps I need to follow the prompts and persist with running `bin/install-wp-tests.sh` and whatever it needs (`svn` ...).

Ideally we'd still have this documented, even if only with one specific dev environment.

Reply:
Yeah `vvv` I don't think required special setup. These tests are also ran by Travis FWIW.

Reply:
It sounds like you have a custom docker environment setup @haszari ?  If you like docker, you might want to consider something like the [`wp env` package](https://github.com/WordPress/gutenberg/tree/master/packages/env) to setup your dev environment with.  At a minimum, even if you don't want to use it directly, you should be able to derive from it's configuration how to setup phpunit for testing with (looks like the `docker-compose.yml` is generated via [this script](https://github.com/WordPress/gutenberg/blob/master/packages/env/lib/create-docker-compose-config.js) there)

Reply:
Thanks for the tip @nerrad I'll give that a try. My docker setup is custom but builds on the [docker wordpress image](https://hub.docker.com/_/wordpress/). I might just need to `apt-get install svn` and then the script (`install-wp-tests.sh`) will work. From quick googling it looks like VVV comes with svn preinstalled.